### PR TITLE
Address issues in readonly reference

### LINKED ISSUE #27148: Missing info on reradonly properties
[Enter feedback here]

There is no explanation how the readonly keyword affects properties. Not ref readonly, just readonly.
---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 26899b81-d6f9-534a-83ee-283dd8aae545
* Version Independent ID: 3061125d-7de4-97ef-ccb0-ee792d6f2ae1
* Content: [readonly keyword - C# Reference](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/readonly)
* Content Source: [docs/csharp/language-reference/keywords/readonly.md](https://github.com/dotnet/docs/blob/main/docs/csharp/language-reference/keywords/readonly.md)
* Product: **dotnet-csharp**
* Technology: **csharp-language-reference**
* GitHub Login: @BillWagner
* Microsoft Alias: **wiwagn**

### LINKED ISSUE #38550: Readonly can be applied to a getter
### Type of issue

Missing information

### Description

`readonly` can be applied to getters, but this information is missing from the article. https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/struct#readonly-instance-members has this information.

### Page URL

https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/readonly

### Content source URL

https://github.com/dotnet/docs/blob/main/docs/csharp/language-reference/keywords/readonly.md

### Document Version Independent Id

3061125d-7de4-97ef-ccb0-ee792d6f2ae1

### Article author

@BillWagner

### Metadata

* ID: 26899b81-d6f9-534a-83ee-283dd8aae545 
* Product: **dotnet-csharp**
* Technology: **csharp-language-reference**

---
[Associated WorkItem - 189972](https://dev.azure.com/msft-skilling/Content/_workitems/edit/189972)

### PR DESCRIPTION
**Pull request by BillWagner**
First commit:

Fixes #38550 - Add a note that `readonly` can be applied to a `get` accessor of a property.

Fixes #27148 - One example had already been added in #37250. I added a link to the pertinent section of the `struct` types.

Finally, do a grammar check.

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [docs/csharp/language-reference/keywords/readonly.md](https://github.com/dotnet/docs/blob/618aaec324cba4ffe82f9c18a3f88e1ac6c71577/docs/csharp/language-reference/keywords/readonly.md) | [readonly (C# Reference)](https://review.learn.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/readonly?branch=pr-en-us-39016) |

### CODE REVIEW

**Comment by BillWagner** [comment]
Ping @adegeo for review, as you wrote one of the issues addressed.

File: docs/csharp/language-reference/keywords/readonly.md [hunk 1]
```diff
@@ -72,17 +72,22 @@ You can also use the `readonly` modifier to declare that an instance member does
...
 
-[!code-csharp[readonly return example](snippets/ReadonlyKeywordExamples.cs#ReadonlyReturn)]
+:::code language="csharp" source="snippets/ReadonlyKeywordExamples.cs" id="ReadonlyReturn":::
 
 The type returned doesn't need to be a `readonly struct`. Any type that can be returned by `ref` can be returned by `ref readonly`.
 
 ## Readonly ref readonly return example
 
-Ref readonly return can also be used in conjunction with `readonly` instance members on `struct` types:
+Ref readonly return can also be used with `readonly` instance members on `struct` types:
```
Comment (tdykstra) [inline docs/csharp/language-reference/keywords/readonly.md:90]: Should this be `ref read only return` code style?

**Review by tdykstra — APPROVED** [review]
LGTM, one nit question.

File: docs/csharp/language-reference/keywords/readonly.md (hunk 1 above)
Comment (BillWagner) [inline docs/csharp/language-reference/keywords/readonly.md:90]: Yes:

```suggestion
A `ref readonly return` can also be used with `readonly` instance members on `struct` types:
```

**Pull request merged**